app/domain/errors.rb

```diff
@@ -485,61 +485,61 @@ module AuthnJwt
-        msg: "Failed to parse mandatory claims, mandatory claims value is empty, or was not found.",
+        msg:  "Failed to parse mandatory claims, mandatory claims value is empty, or was not found.",
         code: "CONJ00106E"
       )
 
       InvalidMandatoryClaimsFormat = ::Util::TrackableErrorClass.new(
-        msg: "mandatory claims value '{0-mandatory-claims-value}' is in invalid format. " \
-             "The format should be claim names separated with comma.",
+        msg:  "mandatory claims value '{0-mandatory-claims-value}' is in invalid format. " \
+              "The format should be claim names separated with comma.",
         code: "CONJ00107E"
       )
 
       InvalidMandatoryClaimsFormatContainsDuplication = ::Util::TrackableErrorClass.new(
-        msg: "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
+        msg:  "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
         code: "CONJ00108E"
       )
 
       MappingClaimsMissingInput = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, the mapping claims value is empty, or was not found.",
+        msg:  "Failed to parse mapping claims, the mapping claims value is empty, or was not found.",
         code: "CONJ00109E"
       )
 
       MappingClaimsBlankOrEmpty = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, one ore more mapping statements is blank or empty " \
-             "'{0-mapping-claims-value}'.",
+        msg:  "Failed to parse mapping claims, one ore more mapping statements is blank or empty " \
+              "'{0-mapping-claims-value}'.",
         code: "CONJ00110E"
       )
 
       MappingClaimInvalidFormat = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, the mapping claim value '{0-mapping-claim-value}' is in invalid format."\
-             "The format should be 'annotation_name:claim_name'",
+        msg:  "Failed to parse mapping claims, the mapping claim value '{0-mapping-claim-value}' is in invalid format."\
+              "The format should be 'annotation_name:claim_name'",
         code: "CONJ00111E"
       )
 
       MappingClaimInvalidClaimFormat = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, one of the claims in mapping claim value '{0-mapping-claim-value}' " \
-             "is in invalid format : {1-claim-verification-error}.",
+        msg:  "Failed to parse mapping claims, one of the claims in mapping claim value '{0-mapping-claim-value}' " \
+              "is in invalid format : {1-claim-verification-error}.",
         code: "CONJ00112E"
       )
 
       MappingClaimDuplicationError = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, {0-purpose} value '{1-claim-value}' appears twice or more",
+        msg:  "Failed to parse mapping claims, {0-purpose} value '{1-claim-value}' appears twice or more",
```

Comment:
please rpovide filled in example

Reply:
```suggestion
        msg:  "Failed to parse mapping claims, {0-purpose} value '{1-claim-value}' appears more than once",
```

Reply:
Done

Reply:
purpose is can be `annotation` or `claim`
claim value is a single value like `branch`
the error will be thrown when mapping claims value is like:
- "**branch**:ref, **branch**:kuku"
or
- "branch:**ref**, namespace:**ref**"